contribs/gmf/src/authentication/service.js

```diff
@@ -0,0 +1,242 @@
+goog.provide('gmf.authentication.service');
+
+goog.require('ngeo.CustomEvent');
+goog.require('gmf');
+goog.require('ol.events.EventTarget');
+
+// todo change test
+
+
+/**
+ * @typedef {{
+ *     functionalities: (gmfx.AuthenticationFunctionalities|undefined),
+ *     is_password_changed: (boolean|undefined),
+ *     role_id: (number|undefined),
+ *     role_name: (string|undefined),
+ *     username: (string|undefined)
+ * }}
+ */
+gmf.authentication.AuthenticationLoginResponse;
+
+
+/**
+ * @typedef {{
+ *     success: boolean
+ * }}
+ */
+gmf.authentication.AuthenticationDefaultResponse;
+
+
+/**
+ * @enum {string}
+ */
+gmf.authentication.AuthenticationRouteSuffix = {
```

Comment:
If it's in the gmf.authentification.Service file (and used by it), it should be prefixed like that I think:
`gmf.authentication.service.AuthenticationRouteSuffix = {...}`
Or even `gmf.authentication.service.RouteSuffix = {...}`